library/common-library/templates/_security-context.tpl

```diff
@@ -0,0 +1,33 @@
+{{- /* Defines the container securityContext context */ -}}
+{{- define "common.securityContext.container" -}}
```

Comment:
I like the idea of `.container` and `.pod` but I think the standard goes more towards the `podSecurityContext` and `containerSecurityContext` direction ([e.g.](https://github.com/kubernetes/ingress-nginx/blob/fb72fcd81772fb3eca923897aec2d92fa5bdff41/charts/ingress-nginx/values.yaml#L811)).

Do we have a strong reason for breaking the "standard"?

Reply:
No. I'm on it.

Reply:
the variable were not the same, it was just the function definition named like that to group them, but yeah better to stick to standards